SDK/Utils/Extensions/DateExtension.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  DateExtension.swift
+//  iOS-SDK
+//
+//  Created by Alberto Huerdo on 6/21/19.
+//  Copyright © 2019 SchedJoules. All rights reserved.
+//
+
+import Foundation
+
+extension Date {
+    
+    func remainingTimeString() -> String {
+        let components = Calendar.current.dateComponents([.day, .hour, .minute], from: self, to: Date())
```

Comment:
Have you considered using a standard component for this? (see https://stackoverflow.com/questions/35999022/how-to-format-time-intervals-for-user-display-social-network-like-in-swift)

Reply:
Here the standard components of calendar is used, the difference with the answer in the links is that here we get more liberty to show a full readable text that can make a difference between showing 1 day and not 1 days and the answer mode of 1d

Reply:
I'll explore other standard options that could also help with localization.

Reply:
Done